infrastructure/gcp/environments/preview/backend.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  backend "gcs" {
+    bucket = "PROJECT_ID-tfstate"
+    prefix = "env/preview"
+  }
+}
+ 
```

Comment:
Hey, probably some whitespace that is causing github to say there is no new line.